src/main/java/com/dumbdogdiner/stickyapi/common/util/MathUtil.java

```diff
@@ -0,0 +1,191 @@
+/**
+ * Copyright (c) 2021 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.common.util;
+
+import org.bukkit.Location;
+import org.bukkit.util.Vector;
```

Comment:
No bukkit depends in common please.

Reply:
Sorry! Would suggest a bukkit-only LocationUtil at some point? Moved one required offset() method to PlayerSelector

Reply:
We can shove in some vector methods into common if we ever need to - could use them for client-side particle spawning on bungee's end too.